### PR TITLE
Update doc to reflect that PG now supports Coq-generated proof diffs

### PR DESCRIPTION
**Pull request by jfehrle**
**Kind:** documentation

Per https://github.com/ProofGeneral/PG/pull/421#issuecomment-496040930, the necessary PG code was merged and is available through MELPA.  Updated PG doc will be posted online by May 31.  No new version number for PG just yet.

Hope we can get this into 8.10.

- [x] Corresponding documentation was added / updated (including any warning and error messages added / removed / modified).
- [x] Entry added in the changelog (see https://github.com/coq/coq/tree/master/doc/changelog#unreleased-changelog for details).

### CODE REVIEW

**Comment by jfehrle** [comment]
@Zimmi48, @cpitclaudel: Could one of you review this very short doc PR soon?  I think it's worth getting into 8.10.  Thanks.

@vbgl If agreeable to the reviewers, can we get this doc-only PR into 8.10?

**Review by cpitclaudel — APPROVED** [review]
LGTM. I'll leave it to the release manager to merge this where appropriate.
Thanks for the ping, too.

**Review by Zimmi48 — CHANGES_REQUESTED** [review]
Please change the location of the changelog entry and I'll merge.

File: doc/sphinx/changes.rst
Comment (jfehrle) [reply doc/sphinx/changes.rst]: Handling it that way seems strange since #10019 was in the beta, although https://github.com/ProofGeneral/PG/pull/421 wasn't done at beta time.  How does the new process handle corrections to the changelist info (e.g. a major omission or something got listed that didn't make the release)?

No problem moving it if you still think that's best.

**Comment by jfehrle** [comment]
Updated.

File: doc/sphinx/proof-engine/proof-handling.rst
Comment (erikmd) [inline doc/sphinx/proof-engine/proof-handling.rst]:
```suggestion
"`Showing Proof Diffs" <https://proofgeneral.github.io/doc/userman/Coq-Proof-General/#Showing-Proof-Diffs>`_)
```

Ref.: https://github.com/ProofGeneral/PG/pull/421#issuecomment-497982497

Comment (erikmd) [reply doc/sphinx/proof-engine/proof-handling.rst]: @jfehrle @cpitclaudel As mentioned in ProofGeneral/PG#421 I've rewritten the URL of all PG's online doc to have more stable URLs.

Comment (cpitclaudel) [reply doc/sphinx/proof-engine/proof-handling.rst]: Sounds good. Should we add redirects, or do we have sufficiently few links?

**Comment by jfehrle** [comment]
I updated the URL and repushed.

File: doc/sphinx/proof-engine/proof-handling.rst
Comment (erikmd) [reply doc/sphinx/proof-engine/proof-handling.rst]: Hi @cpitclaudel,
According to [this search](https://github.com/search?q=https%3A%2F%2Fproofgeneral.github.io%2Fdoc%2Fuserman%2F&type=Code) there are a few such links for `userman` in GitHub, so I've just added redirects using the [jekyll-redirect-from](https://blog.webjeda.com/jekyll-redirect/) plugin, for both `userman` and `adaptingman`.
(cf. the corresponding [commits](https://github.com/ProofGeneral/proofgeneral.github.io/compare/99779ab2ce11cff12b674dafed20ea65f01eeeff...b680be021896d9e3cb83700265ef1880654e5b55))

Comment (cpitclaudel) [reply doc/sphinx/proof-engine/proof-handling.rst]: Super

**Comment by jfehrle** [comment]
@Zimmi48 I think ready to merge.

File: doc/changelog/12-misc/10019-PG-proof-diffs.rst
Comment (Zimmi48) [inline doc/changelog/12-misc/10019-PG-proof-diffs.rst]: I think this parenthesis is not interesting to the end user and could be removed.

**Review by Zimmi48 — APPROVED** [review]

**Comment by jfehrle** [comment]
Removed one comment as you suggested.  Should be good to go now.

**Comment by jfehrle** [comment]
@vbgl Can we also get this into 8.10 now that it's in master?  Thanks.

**Pull request merged**
